flink-connector-cassandra/pom.xml

```diff
@@ -0,0 +1,423 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.flink</groupId>
+		<artifactId>flink-connector-cassandra-parent</artifactId>
+		<version>3.0-SNAPSHOT</version>
+	</parent>
+
+	<groupId>org.apache.flink</groupId>
+	<artifactId>flink-connector-cassandra_${scala.binary.version}</artifactId>
+	<version>3.0-SNAPSHOT</version>
+	<name>Flink : Connectors : Cassandra</name>
+	<packaging>jar</packaging>
+
+	<!-- Allow users to pass custom connector versions -->
+	<properties>
+		<flink.version>1.16.0</flink.version>
+		<flink.shaded.version>15.0</flink.shaded.version>
+		<junit4.version>4.13.2</junit4.version>
+		<junit5.version>5.8.1</junit5.version>
+		<assertj.version>3.21.0</assertj.version>
+		<archunit.version>0.22.0</archunit.version>
+		<testcontainers.version>1.17.2</testcontainers.version>
+		<mockito.version>2.21.0</mockito.version>
+		<slf4j.version>1.7.36</slf4j.version>
+		<log4j.version>2.17.2</log4j.version>
+		<japicmp.skip>false</japicmp.skip>
+		<japicmp.referenceVersion>1.15.0</japicmp.referenceVersion>
+		<scala.binary.version>2.12</scala.binary.version>
+		<byte-buddy.version>1.12.19</byte-buddy.version>
+		<findbugs.version>1.3.9</findbugs.version>
+
+
+		<!--driver 3.x works with 3.x and 4.x versions of Cassandra but driver 4.x is a complete refactoring with different API-->
+		<driver.version>3.11.2</driver.version>
+		<guava.version>19.0</guava.version>
+		<scala-library.version>2.12.7</scala-library.version>
```

Comment:
lets move this closer to scala.binary.version

Reply:
:+1:

Reply:
ah ok, I though you were talking about minor versions :smile: